geode-core/src/test/java/org/apache/geode/internal/cache/DiskStoreImplIntegrationTest.java

```diff
@@ -73,13 +75,10 @@ public void cleansUpOrphanedBackupFilesOnDiskStoreCreation() throws Exception {
     }
 
     cache.close();
-    cache = new CacheFactory(null).create();
+    cache = new CacheFactory().create();
```

Comment:
Why is it okay to not set MCAST_PORT to "0" here?
I think it would be better if you just had a private method that will create the cache and you could call that method in both setup and here.

Reply:
Good point: I will make that change.